src/features/CurrentSessionView/currentSessionBills/CurrentSessionBills.Screen.js

```diff
@@ -1,18 +1,123 @@
-import React from 'react';
-import { View, Text, StyleSheet, Button } from 'react-native';
+import React, { useContext, useEffect, useState } from 'react';
+import { View, Text, StyleSheet, TouchableOpacity, FlatList } from 'react-native';
+import { colors } from '../../../infrastructure/colors.js';
+import { BiteShareContext } from '../../../BiteShareContext.js';
+import { getADocReferenceFromCollection, readASingleDocument } from '../../../../firebase/helpers/database.firebase.js';
+import OrderedItemInfo from './OrderedItemInfo.js';
+import TipInfo from './TipInfo.js';
+
 
 const styles = StyleSheet.create({
   billsContainer: {
     flex: 1,
-    justifyContent: 'center',
+    justifyContent: 'flex-start',
     alignItems: 'center',
+  },
+
+  orderedItemsContainer: {
+    height: 200,
+    backgroundColor: colors.brand.kazanLight2,
+    margin: 15,
+    marginTop: 40,
+    padding: 40,
+    paddingTop: 20,
+    paddingBottom: 20,
+    borderRadius: 40
+  },
+
+  priceContainer: {
+    alignItems: 'center',
+    margin: 50,
+    marginTop: 20,
+    marginBottom: 35
+  },
+
+  individualBill: {
+    paddingBottom: 10,
+    color: colors.brand.rausch,
+    fontWeight: 'bold'
+  },
+
+  paymentButton: {
+    width: 150,
+    marginBottom: 40,
+    padding: 15,
+    borderRadius: 50,
+    backgroundColor: colors.brand.ebisuLight,
+    borderColor: colors.brand.beach,
+    borderWidth: 1
   }
 });
 
 const CurrentSessionBills = ({ changeTab }) => {
+
+  const { state: { accountHolderName, sessionId, orderedItems }, dispatch } = useContext(BiteShareContext);
+  const [individualBill, setIndividualBill] = useState(0);
+  const [totalBill, setTotalBill] = useState(0);
+  const tipPercentages = [0.1, 0.15, 0.2];
+
+  //function to add tax- 7.25%
+  const addTax = (bill) => (bill * .0725) + bill;
+
+  const getIndividualBill = () => {
+    getADocReferenceFromCollection(`transactions/${sessionId}/attendees`, 'name', '==', accountHolderName)
```

Comment:
```suggestion
    getADocReferenceFromCollection(`transactions/${sessionId}/attendees`, 'name', '==', nickname || accountHolderName)
```

When a user has a `nickname` in our app their nickname will be stored into the DB, so when `accountHolderName !== nickname` the individual bill is not updated in the UI

Reply:
added nickname || accountHolderName